nemo/collections/asr/modules/conv_asr.py

```diff
@@ -554,7 +555,7 @@ def __init__(
                 )
             )
             self.decoder_layers.append(nn.Conv1d(self.feat_hidden, self.feat_hidden, kernel_size=1, bias=True))
-            self.decoder_layers.append(nn.BatchNorm1d(self.feat_hidden, eps=1e-3, momentum=0.1))
+            self.decoder_layers.append(BatchNorm1dNoAutoCast(self.feat_hidden, eps=1e-3, momentum=0.1))
```

Comment:
Won't this affect pretrained checkpoints ?

Reply:
No they are fine - submodule has same name and parameters.